### DISCUSSION TITLE
Fifteen Bootstrap columns on homepage?

### DISCUSSION THREAD

**Discussion by pensivedog** [thuliteio/doks#872] [Q&A] [answered]
I'm trying to understand why the Hyas (and Doks) homepage sections with three columns are each defined by `col-lg-5`, which adds up to 15 total columns per row, rather than Bootstrap's standard 12 columns.

If I change them simply to `col-4`, then they each become full width rather than three columns, as we'd expect with Bootstrap. 

I'm relatively new to Bootstrap, so my apologies if this is something trivial or a technique I just haven't learned yet.

Can someone explain this, please?

**Comment by h-enk** [comment, marked as answer]
That's right, I've customized Doks (Hyas is using the defaults) to use 16 columns (I thought it better suiting the layout) instead of 12 — see also: https://github.com/h-enk/doks/blob/master/assets/scss/common/_variables.scss#L90-L96 + the Bootstrap docs: [Customizing the grid](https://getbootstrap.com/docs/5.2/layout/grid/#customizing-the-grid)

And, I'm centering the columns in a row using `justify-content-center` (see also: https://github.com/h-enk/doks/blob/master/layouts/index.html#L20-L33) + the Bootstrap docs: [Horizontal alignment](https://getbootstrap.com/docs/5.2/layout/columns/#horizontal-alignment)

To learn more about customizing columns, see the Bootstrap docs: [Columns](https://getbootstrap.com/docs/5.2/layout/columns/)

**Reply by pensivedog** [reply]
Thanks for explaining that!
